data/ME/incentive_relationships.json

```diff
@@ -0,0 +1,8 @@
+{
+  "exclusions": {
+    "ME-1": ["ME-2"],
+    "ME-6": ["ME-7"],
+    "ME-8": ["ME-9"],
```

Comment:
I think these two should be the other way around? ME-6 and ME-8 are the moderate-income versions; instead it should be the low-income version excludes the moderate-income version. (I.e. if we show you the low-income incentive, you shouldn't also see the moderate-income incentive even though your income is necessarily below the moderate thresholds.)

Reply:
Yup that's right, thanks!